include/secp256k1_bulletproofs.h

```diff
@@ -0,0 +1,176 @@
+ *  In:       proof: array of byte-serialized rangeproofs (cannot be NULL)
+ *         n_proofs: number of proofs in the above array, and number of arrays in the `commit` array
+ *             plen: length of every individual proof
+ *        min_value: array of arrays of minimum values to prove ranges above, or NULL for all-zeroes
+ *           commit: array of arrays of pedersen commitment that the rangeproofs is over (cannot be NULL)
+ *        n_commits: number of commitments in each element of the above array (cannot be 0)
+ *            nbits: number of bits in each proof
+ *        value_gen: generator multiplied by value in pedersen commitments (cannot be NULL)
+ *     extra_commit: additonal data committed to by the rangeproof (may be NULL if `extra_commit_len` is 0)
+ *     extra_commit_len: array of lengths of additional data
+ */
+SECP256K1_WARN_UNUSED_RESULT SECP256K1_API int secp256k1_bulletproof_rangeproof_verify_multi(
+    const secp256k1_context* ctx,
+    secp256k1_scratch_space* scratch,
+    const secp256k1_bulletproof_generators *gens,
+    const unsigned char* const* proof,
+    size_t n_proofs,
+    size_t plen,
+    const uint64_t* const* min_value,
+    const secp256k1_pedersen_commitment* const* commit,
+    size_t n_commits,
+    size_t nbits,
+    const secp256k1_generator* value_gen,
+    const unsigned char* const* extra_commit,
+    size_t *extra_commit_len
+) SECP256K1_ARG_NONNULL(1) SECP256K1_ARG_NONNULL(2) SECP256K1_ARG_NONNULL(3) SECP256K1_ARG_NONNULL(4) SECP256K1_ARG_NONNULL(8);
+
+/** Extracts the value and blinding factor from a single-commit rangeproof given a secret nonce
+ *  Returns: 1: value and blinding factor were extracted and matched the input commit
+ *           0: one of the above was not true, extraction failed
+ *  Args:       ctx: pointer to a context object (cannot be NULL)
+ *             gens: generator set used to make original proof (cannot be NULL)
+ *  Out:      value: pointer to value that will be extracted
+ *            blind: pointer to 32-byte array for blinding factor to be extracted
+ *  In:       proof: byte-serialized rangeproof (cannot be NULL)
+ *             plen: length of every individual proof
+ *        min_value: minimum value that the proof ranges over
+ *           commit: pedersen commitment that the rangeproof is over (cannot be NULL)
+ *        value_gen: generator multiplied by value in pedersen commitments (cannot be NULL)
+ *            nonce: random 32-byte seed used to derive blinding factors (cannot be NULL)
+ *     extra_commit: additonal data committed to by the rangeproof
+ * extra_commit_len: length of additional data
+ */
+SECP256K1_WARN_UNUSED_RESULT SECP256K1_API int secp256k1_bulletproof_rangeproof_rewind(
+    const secp256k1_context* ctx,
+    const secp256k1_bulletproof_generators* gens,
+    uint64_t* value,
+    unsigned char* blind,
+    const unsigned char* proof,
+    size_t plen,
+    size_t min_value,
```

Comment:
Think that's slipped through again, should be `uint64_t min_value` (actually found this is causing compilation errors on some setups)

Reply:
Fixed